app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java

```diff
@@ -12,38 +21,64 @@
 
     @Inject LoginPresenter() {}
 
-    void onLoginValidation(String email, String password) {
-        boolean isValidEmail = validateEmail(email);
-        boolean isValidPassword = validatePassword(password);
+    void onLoginValidation(String email, String password, String dbBaseUrl) {
+        boolean isValidEmailFormat = validateEmailFormat(email);
+        boolean isValidPasswordFormat = validatePasswordFormat(password);
 
-        if (isValidEmail && isValidPassword) {
-            getView().onValidatedLogin();
+        if (isValidEmailFormat && isValidPasswordFormat) {
+            Retrofit retrofit = new Retrofit.Builder()
```

Comment:
It's not really necessary to do this, we can Inject retrofit services as I suggested to do with ToastFactory

Reply:
✔ **Done**